packages/valko-ui-components/src/types/Select.ts

```diff
@@ -1,11 +1,11 @@
-import type { DefaultComponent } from './common'
+import type { DefaultComponent, AriaAttributes } from './common'
 
 export interface SelectOption<T = string | number> {
   value: T;
   label: string;
 }
 
-export interface SelectProps extends DefaultComponent {
+export interface SelectProps extends DefaultComponent, Pick<AriaAttributes, 'aria-label' | 'aria-describedby' | 'aria-labelledby' | 'aria-required' | 'aria-invalid'> {
```

Comment:
Check if this is shorter using Omit instead of Pick, since you are picking a lot of props